.circleci/build_dependencies.sh

```diff
@@ -29,8 +29,8 @@ listdeps() {
 
 listdeps > $DEPS_BEFORE
 
-sudo zypper ar -f https://download.opensuse.org/repositories/devel:openQA/openSUSE_Leap_15.2 devel_openQA
-sudo zypper ar -f https://download.opensuse.org/repositories/devel:/openQA:/Leap:/15.2/openSUSE_Leap_15.2 devel_openQA_Leap
+sudo zypper ar --priority 90 -f https://download.opensuse.org/repositories/devel:openQA/openSUSE_Leap_15.2 devel_openQA
+sudo zypper ar --priority 91 -f https://download.opensuse.org/repositories/devel:/openQA:/Leap:/15.2/openSUSE_Leap_15.2 devel_openQA_Leap
```

Comment:
But this is the wrong way around. The version specific Leap subrepos should be preferred in case they provide version specific overrides

Reply:
That's true. I'll fix it.